index.js

```diff
@@ -8,7 +8,10 @@ const cookieParser = require('cookie-parser');
 const app = express();
 
 // setup middleware
-app.use(cors());
+var corsOptions = {
+	origin: 'http://localhost:3000'
+};
+app.use(cors(corsOptions));
```

Comment:
1. why do we use `var` in here?
2. No need to define a variable for the cors config unless we use it elsewhere in the same file